lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -278,14 +278,17 @@ describe('ReportRenderer', () => {
     });
 
     it('renders without axe violations', (done) => {
-      const container = renderer._dom._document.body;
+      const container = renderer._dom._document.createElement('main');
```

Comment:
any particular reason for this change?

Reply:
Yep, there's a bit of history here :)

In short: as best I can tell, the `report-renderer-test` runs `axe-core` directly against the HTML output of `lighthouse`, and since `axe-core@3.5.1` this has begun reporting a failure (ref: [Travis log output line](https://travis-ci.org/GoogleChrome/lighthouse/builds/650795354#L956)).

I've unhidden some previous comments which include my investigation history ([one](https://github.com/GoogleChrome/lighthouse/pull/10344#issuecomment-586694056), [two](https://github.com/GoogleChrome/lighthouse/pull/10344#issuecomment-586744070), [three](https://github.com/GoogleChrome/lighthouse/pull/10344#issuecomment-587213932)).

Reply:
Gotcha thanks for linking those, great investigation! All makes sense now 👍